pkg/sloop/server/internal/config/config.go

```diff
@@ -78,6 +87,15 @@ func registerFlags(fs *flag.FlagSet, config *SloopConfig) {
 	fs.StringVar(&config.ApiServerHost, "apiserver-host", "", "Kubernetes API server endpoint")
 	fs.BoolVar(&config.WatchCrds, "watch-crds", true, "Watch for activity for CRDs")
 	fs.StringVar(&config.RestoreDatabaseFile, "restore-database-file", "", "Restore database from backup file into current context.")
+	fs.Float64Var(&config.BadgerDiscardRatio, "badger-discard-ratio", 0.01, "Badger value log GC uses this value to decide if it wants to compact a vlog file.  Smaller values free more disk space but use more computing resources")
+	fs.DurationVar(&config.BadgerVLogGCFreq, "badger-vlog-gc-freq", time.Minute*5, "Frequency of running badger's ValueLogGC")
+	fs.BoolVar(&config.BadgerVLogGCLoop, "badger-vlog-gc-loop", true, "If set, rung badger ValueLogGC in a loop until it finds no data to free")
+	fs.Int64Var(&config.BadgerMaxTableSize, "badger-max-table-size", 0, "Max LSM table size in bytes")
+	fs.BoolVar(&config.BadgerKeepL0InMemory, "badger-keep-l0-in-memory", true, "Keeps all level 0 tables in memory for faster writes and compactions")
+	fs.Int64Var(&config.BadgerVLogFileSize, "badger-vlog-file-size", 0, "Max size in bytes per value log file")
+	fs.UintVar(&config.BadgerVLogMaxEntries, "badger-vlog-max-entries", 0, "Max number of entries per value log files")
+	fs.BoolVar(&config.BadgerUseLSMOnlyOptions, "badger-use-lsm-only-options", true, "Sets a higher valueThreshold so values would be collocated with LSM tree reducing disk usage")
```

Comment:
I was thinking this is a `new feature` enable/disable switch, it looks to be more limited. Thoughts on adding an all or nothing switch of the new behaviors?

Reply:
Most of the new flags are off by default.  LSMonly appears to be the important fix, but its good to have the others for fine tuning (and for testing compaction).  I dont feel these need an all or nothing option